### PR TITLE
enforce basic token-based auth for create/delete/update APIs

### PR DESCRIPTION
**Pull request by jonhines**

### CODE REVIEW

File: routes/locations.js [hunk 1]
```diff
@@ -3,9 +3,17 @@ const locationsDb = require('../db/locations');
 const { getLatLngByAddress } = require('../geocode/geoLookup');
 
 const defaultRangeInMiles = 10;
-
+const HEADER_AUTH_TOKEN = "X-VAXMA-TOKEN";
```
Comment (johndeyrup) [inline routes/locations.js]: What are x-vaxma tokens?

**Comment by johndeyrup** [comment]
Looks good; I was wondering if we might need a more public facing api key for people doing scraping vs ui for volunteers, but a single api key is fine and if we need a second one we can go from there.

File: routes/locations.js (hunk 1 above)
Comment (jonhines) [reply routes/locations.js]: its just the custom header name i came up to check for the token value.

i dont think there is a "standard" header for this type of thing

Comment (mzagaja) [reply routes/locations.js]: After checking it seems in general if you're not using OAuth2 it seems to be the case that there isn't really a standard here. I think this is fine.

**Pull request closed without merging**